src/converge.js

```diff
@@ -31,10 +34,10 @@ var curryN = require('./curryN');
  */
 module.exports = curryN(3, function(after) {
   var fns = _slice(arguments, 1);
-  return function() {
+  return arity(max(pluck('length', arguments)), function() {
```

Comment:
Probably yes for `curryN`, but one other thing:

```javascript
return arity(max(pluck('length', _slice(arguments, 1))), function() {  // or
return curryN(max(pluck('length', _slice(arguments, 1))), function() {
```

The first function -- the join -- should not count when calculating the length of the returned function.